### ISSUE TITLE
incorrect currency value

### ISSUE THREAD

**Issue by f1-outsourcing** [OpenMage/magento-lts#3846]
### Preconditions (*)
OpenMage LTS 19.4.14

### Steps to reproduce (*)
![bug](https://github.com/OpenMage/magento-lts/assets/13450219/2ff2d2ad-7d88-493c-8861-06889fbc2e50)

### Expected result (*)
should be 10.08 not?

### Actual result (*)
12

**Comment by f1-outsourcing** [comment]
same here
![bug2](https://github.com/OpenMage/magento-lts/assets/13450219/1e4e8f7b-bbad-4c1a-ab97-764b741cde7b)

**Comment by f1-outsourcing** [comment]
Yes I guess I will have to do that when I am going online with this.

**Comment by ADDISON74** [comment]
I will move to the Discussion section until we determine based on all the steps whether or not there is a problem in the correct calculation of fees in relation to exchange rates.
